grug/vm-wasm/tests/tester.rs

```diff
@@ -101,7 +101,7 @@ fn immutable_state() {
     suite
         .send_message_with_gas(
             &mut accounts["sender"],
-            1_000_000,
+            2_000_000,
```

Comment:
Now that we have user pay also for gas consumed in `withhold_fee`, we need to increase this for the test to pass.